developers_chamber/ecs_utils.py

```diff
@@ -13,70 +18,279 @@ def _get_ecs_client(region):
     )
 
 
-def deploy_new_task_definition(cluster, service, task_definition, image, region):
+def register_new_task_definition(task_definition, image, region):
     ecs_client = _get_ecs_client(region)
-    old_task = ecs_client.describe_task_definition(
-        taskDefinition=task_definition,
-        include=[
-            'TAGS',
-        ]
-    )
+    
+    try:
+        old_task_definition = ecs_client.describe_task_definition(
+            taskDefinition=task_definition,
+            include=[
+                'TAGS',
+            ]
+        )
+    except ClientError as ex:
+        raise ClickException(ex)
 
     LOGGER.info('Image: {}'.format(image))
-    LOGGER.info('Old task definition:\n{}'.format(old_task))
+    LOGGER.info('Old task definition ARN: {}'.format(old_task_definition['taskDefinition']['taskDefinitionArn']))
 
-    definition = old_task['taskDefinition']
+    definition = old_task_definition['taskDefinition']
     definition['containerDefinitions'][0]['image'] = image
-    new_definition = {
+    new_task_definition = {
         'family': definition['family'],
         'taskRoleArn': definition['taskRoleArn'],
         'executionRoleArn': definition['executionRoleArn'],
         'networkMode': definition['networkMode'],
         'containerDefinitions': definition['containerDefinitions'],
         'requiresCompatibilities': definition['requiresCompatibilities'],
-        'tags': old_task['tags'],
+        'tags': old_task_definition['tags'],
     }
 
     if 'cpu' in definition:
-        new_definition['cpu'] = definition['cpu']
+        new_task_definition['cpu'] = definition['cpu']
     if 'memory' in definition:
-        new_definition['memory'] = definition['memory']
-
-    new_task = ecs_client.register_task_definition(**new_definition)
-    revision = new_task['taskDefinition']['revision']
-    ecs_client.update_service(
-        cluster=cluster,
-        service=service,
-        taskDefinition='{}:{}'.format(task_definition, revision),
-        forceNewDeployment=True,
+        new_task_definition['memory'] = definition['memory']
+
+    try:
+        response = ecs_client.register_task_definition(**new_task_definition)
+    except ClientError as ex:
+        raise ClickException(ex)
+
+    LOGGER.info('New task definition ARN: {}'.format(response['taskDefinition']['taskDefinitionArn']))
+    return response['taskDefinition']['taskDefinitionArn']
+
+
+def get_task_definition_for_service(cluster, service, region):
+    ecs_client = _get_ecs_client(region)
+    services = ecs_client.describe_services(cluster=cluster, services=[service])
+
+    if len(services['services']) != 1:
+        raise ClickException('Services found: {}'.format(len(services['services'])))
```

Comment:
Nebylo by lepší tu vraátit Found more services: {} + vypsat jejich názvy?

Reply:
+ pokud nebude žádná vypsat, že nebyly nalazena žádná

Reply:
Teoreticky to může najít 1 nebo 0 services. Takže jsem si říkal že ta hláška bude stačit.

Reply:
"No services found" nebo "Found services: 0" mi přijde jako to samý.

Reply:
Zároveň kdyby se stal nějakej podivnej bug na straně AWS a našlo to dvě, tak to zjistíme.

Reply:
níže jsem ti psal u podobné chyby, že bys to mohl využít i tady, ten popis

Reply:
Ok provedu